Loading anaconda3/personal
  Loading requirement: fix_unwritable_tmp fix_setxattr
Error in library(data.table) : there is no package called ‘data.table’
Execution halted
